src/operation-node/operation-node-transformer.ts

```diff
@@ -411,6 +411,7 @@ export class OperationNodeTransformer {
       onCommit: node.onCommit,
       frontModifiers: this.transformNodeList(node.frontModifiers),
       endModifiers: this.transformNodeList(node.endModifiers),
+      selectQuery: node.selectQuery,
```

Comment:
You need transform the the `selectQuery`

Reply:
Thanks, fixed!